pkg/testutil/env/sandbox.go

```diff
@@ -172,20 +175,16 @@ func createServiceAccount(s Sandbox) error {
 }
 
 func startCtrlMgr(t *testing.T, mgr manager.Manager) func() {
-	stop := make(chan struct{})
-	wg := &sync.WaitGroup{}
-	wg.Add(1)
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
@chrisseto this is the only way I could get this to work with the e2e tests.  Do you have a better way?

Reply:
Can you elaborate on what you mean?

According to [the docs](https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.8.3/pkg/manager#Manager) the preferred method of tearing down the manager is to cancel the passed in context.

If you wanted to remove the tear down func, you could use `t.Cleanup` to cancel the context? My only worry here is whether or not `ctrl-c` will interrupt the test. I think `go test` does some magic to make it work, but I'm not certain.